openapi/wallet-address-server.yaml

```diff
@@ -0,0 +1,138 @@
+openapi: 3.1.0
+info:
+  title: Wallet Address API
+  version: '1.4'
+  license:
+    name: Apache 2.0
+    identifier: Apache-2.0
+  description: |-
+    The Wallet Address API is a simple REST API to get basic details about a wallet address.
+  contact:
+    email: tech@interledger.org
+servers:
+  - url: 'https://rafiki.money/alice'
+    description: 'Server for Alice\'s wallet address (ie https://rafiki.money/alice)'
+  - url: 'https://rafiki.money/alice'
+    description: 'Server for Bob\'s wallet address subresources (ie https://rafiki.money/alice)'
```

Comment:
The url and the description don't really match. I think this one should be removed.